kernel-kit/debian-diffconfigs/sid

```diff
@@ -0,0 +1 @@
+bookworm
```

Comment:
sid = bookworm, until the sid kernel needs some adjustment the bookworm one doesn't